### DISCUSSION TITLE
Calculate CPU time that the Prisma Data Platform spent processing

### DISCUSSION THREAD

**Discussion by 0xDeeep** [prisma/prisma#19069] [Q&A] [answered]
As the title suggests, how can I calculate the amount of time PDP spends processing my queries? How can I record and analyse it?

**Comment by nurul3101** [comment]
Hello @0xdeepmehta 👋 

Can you please open a Support Ticket on Prisma Data Platform to get your data usage and to get answers to any queries you might have?

You can Click on Support on Top Right Side:
<img width="1679" alt="Screenshot 2023-05-04 at 12 55 45 PM" src="https://user-images.githubusercontent.com/13049676/236137669-20df856e-ccaa-4739-9e7e-8f0541276db7.png">

**Reply by 0xDeeep** [reply]
Sure thing. But why should I need to go through a support ticket for that? Isn't there any standard API or method for this?

**Reply by nurul3101** [reply, marked as answer]
At the moment, there's no API through which you could export your utilisation data. It's on our team's radar and we do plan to add this capability.

**Reply by nurul3101** [reply]
If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
